packages/react/src/components/Tab/Tab.js

```diff
@@ -161,12 +161,12 @@ export default class Tab extends React.Component {
         tabIndex={-1}
         className={classes}
         onClick={evt => {
-          handleTabClick(index, label, evt);
+          handleTabClick(index, evt, disabled);
           onClick(evt);
         }}
         onKeyDown={evt => {
           this.setTabFocus(evt);
-          handleTabKeyDown(index, label, evt);
+          handleTabKeyDown(index, evt, disabled);
```

Comment:
```suggestion
          if (disabled) {
            return;
          }
```
same as above

Reply:
This was how I originally had it, but it caused this test to fail: 
```
        it('updates selected state when pressing space or enter key', () => {
          firstTab.simulate('keydown', { which: spaceKey });
          expect(wrapper.state().selected).toEqual(0);
          lastTab.simulate('keydown', { which: enterKey });
          expect(wrapper.state().selected).toEqual(1);
        });
```
(Tabs-test.js at line 225)
and moving the disabled check to where it is now made it pass. I can make this change for handleTabClick for sure though! That didn't cause any problems.

Reply:
I'll dig into more why that made it fail, not sure what's going on at this point.

Reply:
yeah it looks like if we `return` before the event handler is called, then we actually don't need to make any changes to `Tabs.js` so the tests will still pass I believe

Reply:
Ah gotcha! That makes more sense, we'll do it that way!